inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningRecommender.java

```diff
@@ -174,36 +187,48 @@ private boolean isAlreadyInCleanList(List<AnnotationObject> cleanRecommendationL
         return flag;
     }
 
-    private void removeRejectedOrSkippedAnnotations()
+    private void removeRejectedOrSkippedAnnotations(boolean filterSkippedRecommendation,
+        Date learnSkippedRecommendationTime)
     {
         List<LearningRecord> records = learningRecordService.getAllRecordsByDocumentAndUserAndLayer
             (annotatorState.getDocument(), annotatorState.getUser().getUsername(), selectedLayer);
         for (List<AnnotationObject> recommendations: listOfRecommendationsForEachToken) {
             recommendations.removeIf(recommendation -> doesContainRejectedOrSkippedRecord(records,
-                recommendation));
+                recommendation, filterSkippedRecommendation, learnSkippedRecommendationTime));
         }
         listOfRecommendationsForEachToken.removeIf(recommendationsList ->
             recommendationsList.size() == 0);
```

Comment:
Using `isEmpty()` is generally considered better than `== 0` to check if a collection is empty.

Reply:
have changed `== 0` to `isEmpty()`